doc/sphinx/source/vulkan.rst

```diff
@@ -0,0 +1,126 @@
+Vulkan driver
+=================
+
+NOTE: This driver is incomplete without an active maintainer. Pull Requests welcomed.
+
+
+
+This driver uses libvulkan and clspv to run OpenCL code on GPU devices via Vulkan API.
```

Comment:
Can you swap these two sentences? First the brief intro then the NOTE (perhaps bolded).